packages/patternfly-4/react-core/src/layouts/Level/Level.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/layouts/Level/level';
+
+export interface LevelProps extends React.HTMLProps<HTMLDivElement> {
+   /** Adds space between children. Options are sm, md or lg */
+   gutter?: boolean; 
```

Comment:
Same comment as gallery PR.  Why are we changing the type of gutter?

Reply:
Like in gallery, I thought we were fixing the gutter prop type in all of these, I'll change them all back.